### PR TITLE
Put billing role assignments dev billing microsoft.billing 2019 10 01 preview

### PR DESCRIPTION
**Pull request by tugupt**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by msft-adrianma — APPROVED** [review]

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 1]
```diff
@@ -4473,6 +4473,52 @@
             }
           }
         }
+      },
+      "put": {
+        "tags": [
+          "BillingRoleAssignments"
+        ],
+        "operationId": "EnrollmentRoleAssignments_Put",
```
Comment (wilcobmsft) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json:4481]: Can we remove "Enrollment" from the naming here? This is a billing account-level scope, which is also used outside of EA.

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 2]
```diff
@@ -4473,6 +4473,52 @@
...
+        "description": "Create or update a billing role assignment.",
+        "x-ms-examples": {
+          "PutEnrollmentAdministratorRoleAssignment": {
+            "$ref": "./examples/PutEnrollmentAdministratorRoleAssignment.json"
```
Comment (wilcobmsft) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json:4485]: Same here - please remove "Enrollment" from any naming for billing account-level APIs.

**Comment by tugupt** [comment]
@NullMDR : All the checks are passing now. please review and approve the PR.

**Comment by NullMDR** [comment]
@tugupt Please confirm breaking change in python sdk
[ChangeLog] **Breaking changes**
[ChangeLog] 
[ChangeLog]   - Model EnrollmentAccount has a new signature
[ChangeLog]   - Model Invoice has a new signature
[ChangeLog]   - Removed operation InvoicesOperations.get_latest
[ChangeLog]   - Removed operation InvoicesOperations.list

**Comment by tugupt** [comment]
@NullMD : The latest run shows no issues on python sdk breaks. Can you take a look at the latest run results?

**Review by PhoenixHe-NV — APPROVED** [review]

**Pull request merged**